web/wp-content/themes/dctx-v2/page.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-s15xwa.php, but found phpcs-scan-S15Xwa.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).